internal/service/datazone/domain_test.go

```diff
@@ -0,0 +1,443 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package datazone_test
```

Comment:
Really nice job on the AccTests! The use of _basic for all required attributes, and then individual tests for each other attribute/feature provide lasting maintainability that is forward compatable.

Reply:
Thank you! 😄